embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/SpanServiceImpl.kt

```diff
@@ -140,6 +140,7 @@ internal class SpanServiceImpl(
     }
 
     companion object {
+        const val MAX_INTERNAL_SPANS_PER_SESSION = 1000
```

Comment:
I think we should bump this limit - network requests will easily push past this, and we allow 1000 per session per domain, IIRC. I'd go 3000 to start off with, and maybe look at existing data to inform this limit?

Reply:
Good point. I bumped this to 5000 & we can adjust from there. Really it's just a backstop to avoid stupid amounts of spans getting logged & in practice I don't think this will be reached for the vast majority of payloads